src/pages/playlists/playlists.css

```diff
@@ -0,0 +1,70 @@
+.sm-main .sm-main-feat-new {
+  align-items: initial;
+}
+
+.sm-main-playlist-top {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.sm-main-heading {
+  font-size: 3rem;
+  font-weight: 800;
+}
+
+.sm-playlist-main {
+  border: 2px solid var(--COLOR-HEADER-FONT-DARK);
+  background: var(--COLOR-BG);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 6rem;
+  padding: 2rem;
+  cursor: pointer;
+  border-radius: 3rem;
```

Comment:
Border radius should be given in px which won't affect the result on changing root fon size.